kubernetes-client/1.3.99.wso2v1/pom.xml

```diff
@@ -0,0 +1,152 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+ ~ Copyright (c) 2005-2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Header should be as,
Copyright (c) 2016, WSO2 Inc. (http://wso2.com) All Rights Reserved.